webpack.config.js

```diff
@@ -343,7 +347,7 @@ module.exports = (_env, argv) => {
                 ...config.plugins,
                 ...getBundleAnalyzerPlugin(analyzeBundle, 'external_api')
             ],
-            performance: getPerformanceHints(perfHintOptions, 40 * 1024)
+            performance: getPerformanceHints(perfHintOptions, 85 * 1024)
```

Comment:
Percentage wise this is an icrease of > 50% but we are talking very small sizes here, and this means we were missing some polyfils here!